src/main/java/teammates/storage/sqlentity/responses/FeedbackMissingResponse.java

```diff
@@ -0,0 +1,36 @@
+package teammates.storage.sqlentity.responses;
+
+import teammates.common.datatransfer.questions.FeedbackTextResponseDetails;
+import teammates.storage.sqlentity.FeedbackQuestion;
+
+/**
+ * Represents a missing response.
+ */
+public class FeedbackMissingResponse extends FeedbackTextResponse {
+
+    private String giverSectionName;
+    private String recipientSectionName;
+
+    protected FeedbackMissingResponse() {
+        // required by Hibernate
+    }
+
+    public FeedbackMissingResponse(
+            FeedbackQuestion feedbackQuestion, String giver,
+            String giverSectionName, String recipient, String recipientSectionName
+    ) {
+        super(feedbackQuestion, giver, null, recipient, null, new FeedbackTextResponseDetails("No Response"));
```

Comment:
Creating this super class seems like a good idea, since it has the single responsibility of representing missing responses. This allows us to not have to create FeedbackResponses which represent missing responses elsewhere.